### PR TITLE
Add cookbook entry for user charges

### PR DESCRIPTION
**Pull request by IAlibay**
We need this to tell folks how to assign user charges & avoid some of the problems we've been having recently.

### CODE REVIEW

File: cookbook/user_charges.ipynb [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+   "metadata": {
+    "editable": true,
+    "raw_mimetype": "",
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "Components are usually by default loaded into OpenFE without pre-existing partial charges. This is particularly true when loading from [SDF and PDB files](?how do I reference another section?).\n",
```
Comment (IAlibay) [inline cookbook/user_charges.ipynb:79]: How do we go about referencing another section?

Comment (IAlibay) [reply cookbook/user_charges.ipynb:79]:
```suggestion
    "Components are usually by default loaded into OpenFE without pre-existing partial charges. This is particularly true when loading from [SDF and PDB files](https://docs.openfree.energy/en/stable/cookbook/loading_molecules.html).\n",
```
Temporary hack

File: cookbook/user_charges.ipynb [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+   "metadata": {
+    "editable": true,
+    "raw_mimetype": "",
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "Components are usually by default loaded into OpenFE without pre-existing partial charges. This is particularly true when loading from [SDF and PDB files](https://docs.openfree.energy/en/stable/cookbook/loading_molecules.html).\n",
```
Comment (IAlibay) [inline cookbook/user_charges.ipynb:79]:
```suggestion
    "Components are usually by default loaded into OpenFE without pre-existing partial charges. This is particularly true when loading from [SDF and PDB files](https://docs.openfree.energy/en/stable/cookbook/loading_molecules.html#Loading-small-molecules).\n",
```

File: cookbook/user_charges.ipynb [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+    "   By default this will default to using the OEChem TK if available, otherwise AmberTools' antechamber will be used.\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ae36cf5-a109-4abb-8427-d2c1015e097d",
+   "metadata": {},
+   "source": [
+    "Alternative partial charge methods can be used. For example one could ask for `am1-mulliken` charges. Please see the [OpenFF documentation](https://docs.openforcefield.org/projects/toolkit/en/stable/) for more information."
```
Comment (IAlibay) [inline cookbook/user_charges.ipynb:245]: Link to Molecule.assign_partial_charges

**Review by dwhswenson — APPROVED** [review]
Calling this one good! (Actually, completely forgot that I'd meant to merge this hours ago.)

**Pull request merged**